lib/lockfile.js

```diff
@@ -320,6 +343,8 @@ onExit(() => {
 });
 
 module.exports.lock = lock;
+module.exports.lock.mtimeChecker = createMtimeChecker();
```

Comment:
The checker must be created per `lock`,  because the user might be passing a custom `fs`. We should assign to the `lock` object here: https://github.com/moxystudio/node-proper-lockfile/blob/43759413e6c1b3176cf7b842e332b3c8f5670a2d/lib/lockfile.js#L228